client/landing/gutenboarding/components/domain-picker/index.tsx

```diff
@@ -157,12 +175,15 @@ const DomainPicker: FunctionComponent< Props > = ( {
 								<p>{ __( 'Free for the first year with any paid plan.' ) }</p>
 							) }
 						</div>
-						<ConfirmButton />
-						<CloseButton
-							className="domain-picker__close-button"
-							onClick={ onClose }
-							tabIndex={ -1 }
-						/>
+						<div className="domain-picker__header-buttons">
+							<CancelButton />
+							<ConfirmButton />
+							<CloseButton
+								className="domain-picker__close-button"
+								onClick={ onClose }
+								tabIndex={ -1 }
+							/>
```

Comment:
I feel like this close button should be removed now. Whatever purpose it served (maybe for screen readers?) is now handled by the cancel button.

Reply:
It's still the only Close button displayed when domain picker is rendered as Popover. 
Also I think having a top-right (x) button it's a good a11y pattern for every overlay even if there is also a button doing the same action.

Reply:
>it's a good a11y pattern for every overlay

Remember that it's an overlay from our point of view but for customer it's not as it covers the whole viewport.

Reply:
Actually this button displayed in the popover view.
<img width="811" alt="Screenshot 2020-05-07 at 12 31 30" src="https://user-images.githubusercontent.com/14192054/81278795-d4383380-905e-11ea-8269-c3d91c0cad34.png">